lib/galaxy/jobs/__init__.py

```diff
@@ -192,6 +192,9 @@ def __parse_job_conf_xml(self, tree):
                             else:
                                 self.handlers[tag] = [id]
 
+        # Must define at least one handler to have a default.
+        if not self.handlers:
+            raise ValueError("Job configuration file defines no valid handler elements.")
```

Comment:
Looking better at the whole file, I think `ValueError` is fine here, it's the `config_exception()` function that should raise `ConfigurationError` at https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/jobs/__init__.py#L101 . Is that also something you plan to use in planemo?